styles.css

```
@@ -0,0 +1,3 @@
+@tailwind base;
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@tailwind`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)